documents/transfers.md

```diff
@@ -0,0 +1,691 @@
+
+### Reverse transfers from all linked accounts
+
+```rb
+transferId = "trf_EAznuJ9cDLnF7Y"
+
+para_attr = {
+    "amount":100
+}
+
+Razorpay::Transfer.fetch(transferId).reverse(para_attr)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| transferId*   | string      | The id of the transfer to be fetched  |
+| amount   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
+
+**Response:**
+```json
+{
+  "id": "rvrsl_EB0BWgGDAu7tOz",
+  "entity": "reversal",
+  "transfer_id": "trf_EAznuJ9cDLnF7Y",
+  "amount": 100,
+  "fee": 0,
+  "tax": 0,
+  "currency": "INR",
+  "notes": [],
+  "initiator_id": "CJoeHMNpi0nC7k",
+  "customer_refund_id": null,
+  "created_at": 1580456007
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Hold settlements for transfers
+```rb
+
+paymentId = "pay_EB1R2s8D4vOAKG" 
+
+para_attr = {
+  "transfers": [
+    {
+      "amount": 100,
+      "account": "acc_I0QRP7PpvaHhpB",
+      "currency": "INR",
+      "on_hold": 1
+    }
+  ]
+}
+
+Razorpay::Payment.fetch(paymentId).transfer(para_attr)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| transferId*   | string      | The id of the transfer to be fetched  |
+| transfers   | array     | All parameters listed here https://razorpay.com/docs/api/route/#hold-settlements-for-transfers are supported |
+
+**Response:**
+```json
+{
+  "entity": "collection",
+  "count": 1,
+  "items": [
+    {
+      "id": "trf_EB1VJ4Ux4GMmxQ",
+      "entity": "transfer",
+      "source": "pay_EB1R2s8D4vOAKG",
+      "recipient": "acc_CMaomTz4o0FOFz",
+      "amount": 100,
+      "currency": "INR",
+      "amount_reversed": 0,
+      "notes": [],
+      "fees": 1,
+      "tax": 0,
+      "on_hold": true,
+      "on_hold_until": null,
+      "recipient_settlement_id": null,
+      "created_at": 1580460652,
+      "linked_account_notes": [],
+      "processed_at": 1580460652
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Modify settlement hold for transfers
+```rb
+paymentId = "pay_EAeSM2Xul8xYRo"
+
+para_attr = {
+  "on_hold": "1",
+  "on_hold_until": "1679691505"
+}
+Razorpay::Transfer.fetch(paymentId).edit(para_attr)
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| paymentId*   | string      | The id of the payment to be fetched  |
+| transfers   | array     | All parameters listed here https://razorpay.com/docs/api/route/#hold-settlements-for-transfers are supported |
```

Comment:
on_hold_until is missing

Reply:
added on_hold and on_hold_until param

Reply:
@ankitdas13

Reply:
https://github.com/razorpay/razorpay-ruby/blob/update_transfers/documents/transfers.md#modify-settlement-hold-for-transfers